packages/js-dpp/lib/identity/validation/validatePublicKeysFactory.js

```diff
@@ -38,17 +40,24 @@ function validatePublicKeysFactory(validator, bls) {
    * @typedef validatePublicKeys
    *
    * @param {RawIdentityPublicKey[]} rawPublicKeys
+   * @param {boolean} [mustBeEnabled]
    *
    * @return {ValidationResult}
    */
-  function validatePublicKeys(rawPublicKeys) {
+  function validatePublicKeys(rawPublicKeys, mustBeEnabled = false) {
     const result = new ValidationResult();
 
+    const clonedSchema = lodashCloneDeep(publicKeySchema);
+
+    if (mustBeEnabled) {
+      delete clonedSchema.properties.disabledAt;
+    }
+
     // Validate public key structure
     rawPublicKeys.forEach((rawPublicKey) => {
       result.merge(
         validator.validate(
-          publicKeySchema,
+          clonedSchema,
```

Comment:
The cloned schema should have a different `$id` otherwise it will be conflicts in AJV cache. Look at how we do it when we clone contracts